vmdb/app/models/user.rb

```diff
@@ -915,6 +915,11 @@ def self.current_userid=(userid)
     Thread.current[:userid] = userid
   end
 
+  def self.current_user=(user)
+    Thread.current[:user]   = user
+    Thread.current[:userid] = user.try(:userid)
+  end
```

Comment:
good call. It is so easy to break this stuff.

I ended up going away from this. so will remove